packages/api-explorer/src/scenes/MethodTagScene/MethodTagScene.spec.tsx

```diff
@@ -29,16 +29,28 @@ import { screen, waitFor } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
 
 import { api } from '../../test-data'
-import { renderWithRouter } from '../../test-utils'
+import { renderWithRouterAndReduxProvider } from '../../test-utils'
 import { MethodTagScene } from './MethodTagScene'
 
 const opBtnNames = /ALL|GET|POST|PUT|PATCH|DELETE/
```

Comment:
Would be better to have these opBtnNames (also hard-coded in `TypeTagScene.spec.tsx`) in a util file and use for validating?